javascript/javascript1/week1/task4.js

```diff
@@ -0,0 +1,11 @@
+console.log('Ez Namey!') // Ez Namey (Startup name generator)
+
+let firstWords = ['Amusing', 'Enjoyable', 'Entertaining', 'Lively', 'Pleasant', 'Boisterous', 'Convivial', 'Merry', 'Hyggelig', 'Zippy']
+
+let secondWords = ['Gargle', 'Thingy', 'Master', 'Socks', 'Whiskers', 'Cake', 'Fault', 'Ideas', 'Lettuce', 'Story'];
+
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
const randomNumber = Math.floor(Math.random() * 10) instead of 10 you can use length of array(i.e *firstWords.length).this gives the value according to length.